deploy/helm/sumologic/values.yaml

```diff
@@ -221,29 +217,43 @@ prometheus-operator:
   prometheus:
     additionalServiceMonitors:
       - name: collection-sumologic
-        additionalLabels: 
+        additionalLabels:
           app: sumologic
-          release: prometheus-operator
+          release: collection
```

Comment:
We will also need to document to remind user to override all these hard corded value if they specify a different release name cc @maimaisie